src/main/java/tests/ShopServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ShopServiceTest {
+
+    private ShopService shopService;
+    private List<FruitTransaction> transactions;
+
+    @BeforeEach
```

Comment:
The '@BeforeEach' setup method is good for initializing objects that are common to all tests, but be cautious about the transactions list. It might be better to initialize it in each test method to avoid tests affecting each other due to shared state.